README.md

```diff
@@ -11,12 +11,46 @@ For funksjonalitetsønsker, problemer, og spørsmål kan du [opprette et issue h
 *For feature requests, problems, and questions, please [create an issue here on GitHub][new issue].*
 
 
-Contributing
-------------
-Any help is much appreciated. Take a look at our issues in the [issue tracker], and go crazy.
+## Getting Started
+
+You need *Docker* and *Docker Compose* to get started locally. You may use *Docker* on its own, but it'll require setting the containers up manually.
+
+To get started with a Gunicorn-based development environment with live reloads use `docker-compose`:
+
+```sh
+$ docker-compose up
+```
+
+You'll get a local development environment with SPBM, NGINX and PostgreSQL.
+
+If you'd rather use the built-in Django development server you may do so.
+
+```sh
+$ docker-compose run spbm python manage.py runserver 0.0.0.0:8000
+```
+
+Use `docker-compose` to perform other typical things while developing, such as migrating or loading.
+
+```sh
+$ docker-compose run spbm python manage.py migrate
+$ docker-compose run spbm python manage.py showmigrations --plan
+$ docker-compose run spbm python manage.py loaddata
+$ docker-compose run spbm python manage.py ...

```

Comment:
Burde sikkert legge inn rekkefølgen eller finne en bedre løsning på loaddata for å teste lokalt.

Reply:
Lagt til et reelt eksempel som fungerer. :+1: Legger et ferdig eksempel til med Poetry og *taskipy* i feature/poetry-and-python-3.8.